### PR TITLE
Fixed Label fragment background color mismatch

### PR DESCRIPTION
**Pull request by nervehammer**
For adding default labels, it is better that we first resolve issue #76

### CODE REVIEW

**Comment by imujjwal96** [comment]
It is requested that everytime you make a UI change, submit a Screenshot with your PR

File: app/src/main/java/com/procleus/brime/ui/LabelsFragment.java [hunk 1]
```diff
@@ -28,7 +28,7 @@
 public class LabelsFragment extends Fragment {
 
     private ArrayList<String> labelsRetrieved;
-    private static ListView listView;
+    private  ListView listView;
```
Comment (imujjwal96) [inline app/src/main/java/com/procleus/brime/ui/LabelsFragment.java]: Why is this change required

Comment (nervehammer) [inline app/src/main/java/com/procleus/brime/ui/LabelsFragment.java]: just to fix memory leak and instant run feature ;)

**Comment by nervehammer** [comment]
fixed background
@imujjwal96 please check this.

**Comment by imujjwal96** [comment]
Partly fixes #78
